packages/component-library/src/Sandbox/SandboxBaseMapSelector.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { css } from "emotion";
+
+const baseMapMenu = css(`
+  position: absolute;
+  background: #f3f2f3;
+  padding: 10px;
+  bottom: 0;
+  right: 0;
+  z-index: 2;
+  font-family: "Roboto Condensed", sans-serif;
+`);
+
+const SandboxBaseMapSelector = ({ onBaseMapStyleChange, baseMapStyle }) => {
+  return (
+    <>
+      <div className={baseMapMenu}>
+        <input
+          checked={baseMapStyle === "light"}
+          type="radio"
+          value="light"
+          onChange={onBaseMapStyleChange}
+        />
+        <label for="light">Light</label>
```

Comment:
Great job taking accessibility into account here. Just note that when using JSX to replace `for` with `htmlFor` in labels and to include an `id` with each input. 

I found this page from the React documentation useful when I had to make similar changes to the SandboxToggleSwitch component: https://reactjs.org/docs/accessibility.html#accessible-forms